tests/Aspire.Hosting.Azure.Tests/AzureContainerAppsTests.cs

```diff
@@ -1627,11 +1629,10 @@ public async Task VolumesAndBindMountsAreTranslation()
           "type": "azure.bicep.v0",
           "path": "api.module.bicep",
           "params": {
-            "api_volumes_0_storage": "{api.volumes.0.storage}",
-            "api_volumes_1_storage": "{api.volumes.1.storage}",
-            "api_bindmounts_0_storage": "{api.bindMounts.0.storage}",
-            "outputs_azure_container_apps_environment_default_domain": "{.outputs.AZURE_CONTAINER_APPS_ENVIRONMENT_DEFAULT_DOMAIN}",
-            "outputs_azure_container_apps_environment_id": "{.outputs.AZURE_CONTAINER_APPS_ENVIRONMENT_ID}"
+            "env_outputs_volumes_api_0": "{env.outputs.volumes_api_0}",
```

Comment:
@davidfowl - can you verify the changes to this test are as expected?

Reply:
This is a bug.

Reply:
Fixed.